sympy/combinatorics/perm_groups.py

```diff
@@ -1817,8 +1817,12 @@ def is_normal(self, gr, strict=True):
         True
 
         """
+        if not self.is_subgroup(gr, strict=strict):
+            return False
         d_self = self.degree
         d_gr = gr.degree
+        if self.is_trivial and (d_self == d_gr or not strict):
```

Comment:
It seems that if `self` and `gr` are both trivial, then `d_self == d_gr` is true and therefore `True` will be returned independently of the value of `strict`.

Reply:
I can't follow. If ``self`` and ``gr`` are trivial but ``d_self == d_gr``, then it does not matter if ``strict`` is true or false, as both groups already act on the same number of elements.
If ``d_self != d_gr`` and ``strict==True`` we continue as before.

Maybe there is a misunderstanding what ``strict`` does?

Reply:
Ignore that. I was under the impression that `strict` would mean "strict subgroup".